lib/HTMLAssetWithLazySrc.js

```diff
@@ -6,8 +6,19 @@ class HTMLAssetWithLazySrc extends HTMLAsset {
         this.ast.walk(node => {
             if (node.attrs) {
                 for (let attr in node.attrs) {
-                    if (attr === 'data-src' || attr === 'data-srcset') {
-                        node.attrs[attr] = super.collectSrcSetDependencies(node.attrs[attr]);
+                    if (attr === 'data-src' || attr === 'data-srcset' || attr === 'data-bgset') {
+                        
+                        if (attr === 'data-bgset') {
+                            const uniqueId = `${Date.now()}_${Math.random().toString(36).substring(2)}`;
+                            const separator = ',';
+                            const replacementSeparator = '|';
+                            
+                            const attrVal = node.attrs[attr].replaceAll(replacementSeparator, `_${uniqueId}${separator}`);
+                            const srcSetDependencies = super.collectSrcSetDependencies(attrVal);
```

Comment:
how does parcel handle those separators? why do we need this replacement logic in the first place? is it because parcel can't handle pipes | used by lazysizes?

Reply:
I couldn't find any information in Parcel's documentation about adding an additional delimiter/separator for HTML files. It seems that Parcel only recognizes commas as delimiters. To address this, my proposed solution is to replace the '|' character with a comma, but with a unique identifier as a prefix. Once Parcel finishes processing the code, we can revert the 'comma prefixed with a unique id' back to '|'. I welcome any better suggestions. The unique id prefix ensures accurate identification and prevents unintended replacements of existing commas.

Reply:
Have you tested if Parcel will handle _${uniqueId}${separator} when you pass it to collectSrcSetDependencies?

Reply:
I tested it on a project with numerous background images using the data-bgset attribute. The _${uniqueid}${separator} helps identify which ones to revert back to the '|' separator. I placed the separator after the unique id to prevent unintended issues when the string/code is separated by Parcel.